src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -95,7 +95,9 @@ export class BulkConceptsPage extends Component {
       target: { value },
     } = event;
     if (!value) {
-      this.getBulkConcepts();
+      const page = 1;
```

Comment:
Since you use this once, you can get rid of the constant.

Reply:
Done